### ISSUE TITLE
Jetpack: Settings: Video

### ISSUE THREAD

**Issue by rickybanister** [Automattic/wp-calypso#12507]
WordPress.com is currently completely missing the Video settings card. While we're at it, let's display a video storage indication in the /settings/writing/video card once activated as well as in the media modal (mirroring the space limit for free .com plans). 

![image](https://cloud.githubusercontent.com/assets/437258/24728029/422752c0-1a26-11e7-81cf-3459efec18b5.png)

@dbtlr @MichaelArestad @aheckler @oskosk @tyxla

**Comment by aheckler** [comment]
Once this is done, an HE will need to swing back through https://github.com/Automattic/jetpack/issues/6003 and notify the users.

**Comment by rickybanister** [comment]
That's a 24px margin between the toggle and the planStorage component by the way, but I just dropped it straight in with the inspector and it looks great.

**Comment by rickybanister** [comment]
We will also need to implement the appropriate `banner` for free tier users.

**Comment by MichaelArestad** [comment]
Will the Upgrade button change based on how close it is to being full? (maybe to primary?)

**Comment by rickybanister** [comment]
You can check out the pattern if you like, but I do not believe it changes as the space is used up. Feel free to open a separate issue related to the component.

**Comment by beaulebens** [comment]
Note that for WordPress.com sites, they cannot turn this option off, so we should either have it there, active, and disabled, or perhaps only show the usage meter?

**Comment by rickybanister** [comment]
There is no need for the entire Media card for WordPress.com sites so we may just continue not showing anything. If we do show a media card, we would be displaying the aggregate media upload usage, not just video since we charge for upload space at large.

**Comment by sirreal** [comment]
Would we be interested in rolling this out in 2 PRs? I ask because the functionality is essentially ready and working in #13857, which will let users manage this setting from Calypso _now_, and will show the upgrade nudge when their plan doesn't include video.

The hangup is getting the storage data which doesn't seem to be immediately available in Calypso, so that part needs to be implemented before I can add the storage bar. This could be a feature update after the functionality is rolled out.

---

Now that I'm thinking about the storage bar, Jetpack Professional has unlimited storage. They will either need no storage bar, a report of how much storage they're currently using (no bar), or something else.

**Comment by rickybanister** [comment]
Good catch—I think two PRs is just fine.

And for professional sites, let's use the same style as the bar but no bar—we can say `22g uploaded, unlimited storage available` or something similar. Happy to help with a mockup when you're ready.
